training/src/anemoi/training/diagnostics/mlflow/auth.py

```diff
@@ -217,7 +226,12 @@ def _request(self, path: str, payload: dict) -> dict:
         }
 
         try:
-            response = requests.post(f"{self.url}/{path}", headers=headers, json=payload, timeout=30)
```

Comment:
Making this robust may lead to multiple posts. It looks safe to do here.

Reply:
If I understand corectly, "path" is always "refresh_token" and could (should?) be hard coded here?

Reply:
Yes, it is safe to post multiple times. 

The path is a leftover from when we used to have more than one path. It could be hardcoded, but it's also okay to leave it in case another path ever comes back.